python/cudf/cudf/core/frame.py

```diff
@@ -195,9 +195,41 @@ def _apply_boolean_mask(self, boolean_mask):
         result._copy_categories(self)
         return result
 
+    def _quantiles(
+        self,
+        q,
+        interpolation="LINEAR",
+        is_sorted=False,
+        column_order=[],
```

Comment:
Might be best to use `()` for empty defaults (to avoid using mutable objects here)?

Reply:
```suggestion
        column_order=(),
```